src/AppBundle/Controller/Api/SurveyController.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace AppBundle\Controller\Api;
+
+use AppBundle\Entity\SurveyAnswer;
+use AppBundle\Entity\Survey;
+use AppBundle\Entity\SurveyQuestion;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+class SurveyController extends Controller
+{
+    /**
+     * @Route("/surveys", name="api_surveys")
+     * @Method("GET")
+     */
+    public function apiSurveysAction()
+    {
+        $user = $this->getUser();
+        if (!$user) {
+            return $this->json(['message' => 'User is not authorized'], 401);
+        }
+        $em = $this->getDoctrine()->getManager();
+        $surveys = $em->getRepository(Survey::class)->findSurveyByUser($user);
+        if (!$surveys) {
+            return $this->json(['message' => 'No surveys'], 404);
+        }
+        $serializer = $this->get('serializer');
+
+        $json = $serializer->normalize(
+            $surveys,
+            null,
+            array('groups' => array('group1'))
+        );
+
+        return $this->json($json);
+    }
+
+    /**
+     * @param Survey $survey
+     * @Route("/survey/{id}", name="api_survey_byid")
```

Comment:
well it is could be survey if developer want so.. Need a teacher's advice OVER HERE!!